com.microsoft.mrtk.input/Utilities/FollowJoint.cs

```diff
@@ -19,46 +20,58 @@ namespace Microsoft.MixedReality.Toolkit.Input
     internal class FollowJoint : MonoBehaviour
     {
         [SerializeField]
-        [Tooltip("The hand on which to track the joint.")]
+        [Tooltip("The pose source representing the hand joint this interactor tracks")]
+        private HandJointPoseSource jointPoseSource;
+
+        [SerializeField]
+        [HideInInspector]
+        // A temporary variable used to migrate instnaces of FollowJoint to use the jointPoseSource class as the source of truth
+        // rather than its own separately serialized values.
+        // TODO: Remove this after some time to ensure users have successfully migrated.
+        private bool migratedSuccessfully = false;
+
+
+        [SerializeField]
+        [HideInInspector]
         private Handedness hand;
 
         /// <summary>
         /// The hand on which to track the joint.
         /// </summary>
-        protected Handedness Hand { get => hand; set => hand = value; }
+        [Obsolete("Please change the Hand value on the jointPoseSource instead")]
```

Comment:
Good catch on the access modifiers. I'll go back and change the IPoseSource references to `protected` in the places where the values they were replacing were also labeled as `protected`

Reply:
A bit of an update on this. For `FollowJoint`, I want to leave these properties around, since they allowed developers to set the handedness and joint from code.

For cases like the `PokeInteractor` and the `HandJointInteractor`, I want to remove them because they were accessors that were only used within the class itself. We don't want these interactors to have a notion of "Handedness" built into their main logic, instead deferring that to things like the `HandJointPoseSource`.

Reply:
> I want to leave these properties around, since they allowed developers to set the handedness and joint from code.

If we want to leave these around, should we un-obsolete them?

> I'll still change the pose source instances to protected so that things which subclass our interactors can modify the ways that they are used at least.

For consistency, it might be safer/cleaner to expose `protected` properties (like these!) and leaving the fields `private`.

Reply:
> If we want to leave these around, should we un-obsolete them?

I meant that I wanted to leave those properties but mark them as obsolete because it'd be too breaking to remove them completely currently. I think in the future we want them to modify those properties through the jointPoseSource, which I'll make a `protected` accessible property for!.

Reply:
Refactor done!